src/dashboard/components/Center/component/DashboardHeader.tsx

```diff
@@ -3,13 +3,21 @@ import '../../../../assets/styles/dashboard.scss';
 import { Container, Navbar } from 'react-bootstrap';
 import ToolbarHeader from '../../../../layout/Home/components/Header/Toolbar/component/ToolbarHeader';
 import Toolbarbody from '../../../../layout/Home/components/Header/Toolbar/component/Toolbarbody';
+import { useDateFilterStore } from '../../../../context/useDateFilterStore';
 
 const DashboardHeader = () => {
+  const setDateFilter = useDateFilterStore((state) => state.setDateFilter);
```

Comment:
always name the store as component, to allow using it for other filters than date, please name it `dashboardStore`